packages/cfpb-design-system/src/components/cfpb-tooltips/index.js

```diff
@@ -0,0 +1,79 @@
+/* ==========================================================================
+   Tooltip Organism
+   ========================================================================== */
+
+import tippy from 'tippy.js';
+import { instantiateAll } from '@cfpb/cfpb-design-system';
+
+import * as TooltipStyles from './tooltip.scss';
+
+const BASE_ATTRIBUTE = 'data-tooltip';
+
+/**
+ * Tooltip
+ * @class
+ * @classdesc Initializes a new Tooltip.
+ * @param {HTMLElement} element - The DOM element that should
+ *  activate the tooltip.
+ * @returns {Tooltip} An instance.
+ */
+function Tooltip(element) {
+  const tooltipContent = element.getAttribute(BASE_ATTRIBUTE);
+
+  /**
+   * Set up and create the tooltip.
+   * @returns {Tooltip} An instance.
+   */
+  function init() {
+    tippy(element, {
+      theme: 'cfpb',
+      maxWidth: 450,
+      content: function (reference) {
+        const template = reference.parentElement.querySelector(
+          `#${tooltipContent}`,
+        );
+        const container = document.createElement('div');
+        const node = document.importNode(template.content, true);
+        container.appendChild(node);
+        return container;
+      },
+      // See https://atomiks.github.io/tippyjs/v6/plugins/
+      plugins: [
+        {
+          name: 'hideOnEsc',
+          defaultValue: true,
+          fn({ hide }) {
+            /**
+             * Hide when the escape key is pressed.
+             * @param {KeyboardEvent} event - Key down event.
+             */
+            function onKeyDown(event) {
+              if (event.key === 'Escape') {
+                hide();
+              }
+            }
+            return {
+              onShow() {
+                document.addEventListener('keydown', onKeyDown);
+              },
+              onHide() {
+                document.removeEventListener('keydown', onKeyDown);
```

Comment:
Does it make any sense to target the body with these events. `document.body` instead of `document`?

Reply:
I don't think it really matters. The event will bubble up to both of them. I suppose there could be a situation where the page's `body` isn't filling the entire viewport, in which case pressing escape wouldn't be captured. Although I think the spec says that if no element is focused, it's fired on the `body`. So for clarity I'll just change it to `document.body`.